src/bin/cargo-fmt.rs

```diff
@@ -139,28 +154,113 @@ pub struct Target {
     kind: TargetKind,
 }
 
+#[derive(Debug)]
+pub enum WorkspaceHitlist {
+    All,
+    Some(Vec<String>),
+    None,
+}
+
+impl WorkspaceHitlist {
+    fn is_all(&self) -> bool {
+        match *self {
+            WorkspaceHitlist::All => true,
+            _ => false,
+        }
+    }
+
+    fn is_none(&self) -> bool {
```

Comment:
If you derive `Eq` and `ParitalEq` then you could use `==` rather than needing the `is_all` and `_none` methods